classes/patterns/class-pageheader.php

```diff
@@ -38,44 +38,53 @@ public static function get_name(): string {
 	 * @param array $params Optional array of parameters for the config.
 	 */
 	public static function get_config( $params = [] ): array {
-		$classname         = 'is-pattern-p4-page-header';
+		$classname         = self::get_classname();
```

Comment:
In retrospect, I'm not sure I should have made this change 🤔 Because then the classname ends up being `is-pattern-p4-page-header-img-right`, not sure we want that specificity?

Reply:
Yeah it's also going to break the css rules attached to this classname and this classname being a class of the media/text block itself